From cd72696c785f90d61073338f07cdde2b5ed695f5 Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 2 Oct 2023 12:23:24 +0100
Subject: [PATCH 1/4] FrIO: fix mkdir on windows

---
 src/FrIO.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/FrIO.c b/src/FrIO.c
index 7cc740b..c267776 100644
--- a/src/FrIO.c
+++ b/src/FrIO.c
@@ -34,7 +34,11 @@ int FrIOmkdir(char *dirName)
     closedir(dirp);
     return(0);}
 
+#if (defined(_WIN32) || defined(__WIN32__))
+  err = mkdir(dirName);
+#else
   err = mkdir(dirName, S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
+#endif
      
   return(err);}
 
-- 
2.39.2

