From 20c2e65d8cf927cca611ed49b4075cbffd2b147b Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 2 Oct 2023 12:24:02 +0100
Subject: [PATCH 2/2] python: use sysconfig instead of distutils

---
 Python/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Python/CMakeLists.txt b/Python/CMakeLists.txt
index 13f2b76..97c1297 100644
--- a/Python/CMakeLists.txt
+++ b/Python/CMakeLists.txt
@@ -10,8 +10,8 @@ if (ENABLE_PYTHON)
     # find paths by querying python
     execute_process(
         COMMAND "${Python3_EXECUTABLE}" "-c"
-        "from distutils import sysconfig; import numpy;
-print(sysconfig.get_python_lib(plat_specific=True, prefix=''), end=';');
+        "import numpy, sysconfig;
+print(sysconfig.get_path('platlib', vars={'platbase': '.', 'platlibdir': 'lib'}), end=';')
 print(numpy.get_include(), end=';')
 "
         RESULT_VARIABLE _Python3_RESULT
-- 
2.39.2

