diff --git a/src/FrameL.c b/src/FrameL.c
index 8eca331..63492a5 100644
--- a/src/FrameL.c
+++ b/src/FrameL.c
@@ -14816,7 +14817,7 @@ FrTOCevt *FrTOCevtRead(FrFile *iFile)
     memcpy(evt->GTimeS,       &(GTimeS[j]), nEvent[i]*sizeof(int));
     memcpy(evt->GTimeN,       &(GTimeN[j]), nEvent[i]*sizeof(int));
     memcpy(evt->amplitude, &(amplitude[j]), nEvent[i]*sizeof(float));
-    memcpy(evt->position,   &(position[j]), nEvent[i]*sizeof(position));
+    memcpy(evt->position,   &(position[j]), nEvent[i]*sizeof(FRLONG));
     j += nEvent[i];}
                                     /*----------- free the temporary buffer--*/
  free(names);
